docs/xrpl-wallet/initialization.mdx

```diff
@@ -0,0 +1,328 @@
+}
+
+export const BUTTON_POSITION = {
+  BOTTOM_LEFT: "bottom-left",
+  TOP_LEFT: "top-left",
+  BOTTOM_RIGHT: "bottom-right",
+  TOP_RIGHT: "top-right",
+} as const;
+
+export interface NetworkInterface {
+  /**
+   * Block explorer url for the chain
+   * @example https://ropsten.etherscan.io
+   */
+  blockExplorerUrl: string;
+  /**
+   * Logo url for the base token
+   */
+  logo: string;
+  /**
+   * Name for ticker
+   * @example 'Binance Token', 'Ethereum', 'Matic Network Token'
+   */
+  tickerName: string;
+  /**
+   * Symbol for ticker
+   * @example BNB, ETH
+   */
+  ticker: string;
+  /**
+   * RPC target Url for the chain
+   * @example https://ropsten.infura.io/v3/YOUR_API_KEY
+   */
+  rpcTarget: string;
+  /**
+   * Chain Id parameter(hex with 0x prefix) for the network. Mandatory for all networks. (assign one with a map to network identifier for platforms)
+   * @example 0x1 for mainnet, 'loading' if not connected to anything yet or connection fails
+   * @defaultValue 'loading'
+   */
+  chainId: string;
+  /**
+   * Display name for the network
+   */
+  displayName: string;
+
+  /**
+   * xrpl Network Key
+   */
+  networkKey: string;
+}
+
+export const TORUS_BUILD_ENV = {
+  PRODUCTION: "production",
+  DEVELOPMENT: "development",
+  TESTING: "testing",
+} as const;
+```
+
+</TabItem>
+
+</Tabs>
+
+#### `network`
+
+<Tabs
+  defaultValue="table"
+  values={[
+    { label: "Table", value: "table" },
+    { label: "Interface", value: "interface" },
+  ]}
+>
+
+<TabItem value="table">
+
+| Parameter          | Type     | Description                                                                                                                                     |
+| ------------------ | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| `blockExplorerUrl` | `string` | Block explorer url for the chain                                                                                                                |
+| `logo`             | `string` | Logo url for the base token                                                                                                                     |
+| `tickerName`       | `string` | Name for ticker                                                                                                                                 |
+| `ticker`           | `string` | Symbol for ticker                                                                                                                               |
+| `rpcTarget`        | `string` | RPC target Url for the chain                                                                                                                    |
+| `chainId`          | `string` | Chain Id parameter(hex with 0x prefix) for the network. Mandatory for all networks. (assign one with a map to network identifier for platforms) |
```

Comment:
Mention "0x1" is chainId for xrpl mainnet, "0x2" for testnet and "0x3" for devnet.